apps/website/src/content/docs/flex.mdx

```diff
@@ -1,19 +1,43 @@
 ---
 title: Flex
-description: A utility component to work with CSS flexbox and iTwinUI design tokens.
+description: A utility component that makes it easier to work with CSS flexbox and iTwinUI design tokens.
 thumbnail: #TODO
 group: utilities
 ---
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='flex--basic' />
+## Usage
+
+Wrap `Flex` around the components that should be the children of the flex container. Set the gap using the `gap` prop with a value of an iTwinUI size token (e.g. `x`, `xs`) or any valid css value (e.g. "1rem"). The default `gap` is `xs`.
```

Comment:
Small suggestions. Feel free to commit the suggested change. It does the following:
- Split into two paragraphs and mentions `Flex.Item` explicitly.
- Removes mention of arbitrary CSS values, since it is not recommended.
- Links to size tokens
- Minor formatting nits

```suggestion
The `Flex` component is a generic container that can be wrapped around the items that need to be laid out using CSS flexbox. These items can be used directly or with [`Flex.Item`](#flex-items) for more granular customization.

`Flex` offers a `gap` prop, which can accept a value corresponding to [iTwinUI size tokens](/docs/variables#size) (e.g. `"x"`, `"xs"`). By default, a `gap` of `"xs"` is used.
```

Reply:
I have made this change.